libconsensus/Sealer.h

```diff
@@ -195,6 +204,9 @@ class Sealer : public Worker, public std::enable_shared_from_this<Sealer>
     /// handler
     Handler<> m_tqReady;
     Handler<int64_t> m_blockSubmitted;
+
+    uint64_t m_maxBlockCanSeal;

```

Comment:
plz give some comment for this member

Reply:
done, please refer to https://github.com/FISCO-BCOS/FISCO-BCOS/pull/695/files#diff-7169384e87cdf92ca9a6f534a71de137R206